packages/fast-tooling-react/src/form/utilities/navigation.ts

```diff
@@ -0,0 +1,381 @@
+import { cloneDeep, get, memoize } from "lodash-es";
+import { MemoizedFunction } from "lodash";
```

Comment:
lodash?

Reply:
That was erroring when I tried to import from lodash-es, turns out I don't need it though not sure what that was about.